website/blog/2021-07-26-7.15.0.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "7.15.0 Released: Hack-style pipelines, TypeScript const enums and Rhino target support"
+author: Babel Team
+authorURL: https://twitter.com/babeljs
+date:   2021-07-26 0:00:00
+categories: announcements
+share_text: "Babel 7.15.0 Released"
+---
+
+This release enables top-level await parsing by default (Stage 4 at the [May](https://github.com/tc39/agendas/blob/master/2021/05.md) meeting), enables transforming ergonomic brand checks for private fields by default (Stage 4 at the [July](https://github.com/tc39/agendas/blob/master/2021/07.md) meeting) and includes support for the [Hack-style](https://github.com/js-choi/proposal-hack-pipes) pipeline operator. We also improved our TypeScript support, implementing transform support for `const` enums and namespace aliases, and we expanded our heuristics to add `.displayName` to React components created by `React.createContext()` ([#13501](https://github.com/babel/babel/pull/13501)).
+
+We also introduced a new compiler assumption, [`noIncompleteNsImportDetection`](TODO: Link), to produce a smaller output when compiling ECMAScript modules to CommonJS without worrying about partially initialized namespace imports caused by module cycles.
+
+Additionally, you can now specify [Rhino](https://mozilla.github.io/rhino/) as a compilation target.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.15.0).
+
+<!-- truncate -->
+
+> Mention [recent blog post](https://babeljs.io/blog/2021/05/10/funding-update) and quick funding update?
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://opencollective.com/babel) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## Highlights
+
+### ECMAScript features enabled by default
+
+In the last two meetings, the [top-level `await`](https://github.com/tc39/proposal-top-level-await) and [ergonomic brand checks for private fields](https://github.com/tc39/proposal-private-fields-in-in) proposals reached Stage 4.
+
+```javascript
+import * as db from "database";
+
+await db.connect(); // top-level await
+
+class DBConnector {
+  #password;
+  static isConnector(obj) {
+    return #password in obj; // ergonomic brand checks
+  }
+}
+```
+
+Babel now supports them by default, so you can remove the following plugins from your configuration:
+- `@babel/plugin-syntax-top-level-await`
```

Comment:
cool, can move these in website https://github.com/babel/website/blob/d12f0a40b4bf839ee673fce964d2088743d64d82/website/sidebars.json#L66-L69 and add note to the page like in

> **NOTE**: This plugin is included in `@babel/preset-env`

Reply:
I pushed it to https://github.com/babel/website/pull/2556, since I was doing some related changes.